src/frontend/shortlist/src/api/shortlist.js

```diff
@@ -126,3 +129,25 @@ export class temporaryUpdatePreferences {
       .catch((err) => this.failCb(err));
   }
 }
+
+export class temporaryAddStudent {
+  constructor(payload, successCb, failCb) {
+    this.accountId = payload.accountId;
+    this.addEmail = payload.addEmail;
+    this.successCb = successCb;
+    this.failCb = failCb;
+  }
+  execute() {
+    axios({
+      method: "POST",
+      url: "https://api.shortlist.nyc/account/update",
```

Comment:
This will be a different URL later, right? (A new API call)

Reply:
Yup it is, I should just leave it empty for now